single_tech_samples/databricks/sample4_ci_cd/notebook-python-lib/devops/ci-pipeline.yml

```diff
@@ -0,0 +1,113 @@
+trigger: none
+
+pool:
+  name: Hosted Ubuntu 1604
+
+variables:
+  basePath: 'single_tech_samples/databricks/sample4_ci_cd'
+  pythonVersion: 3.8
+
+stages:
+- stage: UT
+  displayName: Python Unit Tests
+  jobs:
+    - job: UnitTest
+      displayName: Run Unit tests
+      pool:
+        vmImage: ubuntu-latest
+      steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: '3.8'
+          architecture: 'x64'
+
+      - task: CopyFiles@2
+        inputs:
+          SourceFolder: '$(basePath)/notebook-python-lib/notebooks'
+          contents: '**'
+          targetFolder: $(Build.ArtifactStagingDirectory)/notebooks
+        displayName: 'Copy notebooks folder'
+
+      - task: CopyFiles@2
+        inputs:
+          SourceFolder: '$(basePath)/notebook-python-lib/tests/integration'
+          contents: '**'
+          targetFolder: $(Build.ArtifactStagingDirectory)/tests/integration
+          displayName: 'Copy tests folder'
+
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: '$(pythonVersion)'
+          addToPath: true
+          architecture: 'x64'
+        displayName: 'Use Python Version: $(pythonVersion)'
+
+      - script: |
+          cd $(basePath)/notebook-python-lib
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+        displayName: 'Install libraries and dependencies'
+
+      - script: |
+          echo "y
+          $(databricksDomain)
+          $(databricksPat)
+          $(databricksClusterId)
+          $(databricksWorkspaceOrgId)
+          15001" | databricks-connect configure
```

Comment:
What does 15001 represent?

Reply:
It is the port number in which the databricks connection is configured.
Will convert that into a variable.

Reply:
Its the port where databricks-connect connects to in the cluster. Maps to `spark.databricks.service.port`.

Reply:
But yeah, perhaps add a comment saying its the Port ?